src/renderer/opengl/BackgroundImageProgram.zig

```diff
@@ -0,0 +1,121 @@
+/// The OpenGL program for rendering terminal cells.
+const BackgroundImageProgram = @This();
+
+const std = @import("std");
+const gl = @import("opengl");
+
+pub const Input = extern struct {
+    /// vec2 terminal_size
+    terminal_width: u32 = 0,
+    terminal_height: u32 = 0,
+
+    /// uint mode
+    mode: BackgroundMode = .aspect,
+};
```

Comment:
This should be moved up a directory level so that I can be shared with the Metal renderer.